scripts/trigger-circles-tx.sh

```
@@ -0,0 +1,225 @@
+set -e
+
+# Ensure the seed phrase is available
+if [[ -z "$CIRCLES_SEED_PHRASE" ]]; then
+    echo "Error: CIRCLES_SEED_PHRASE is not set." >&2
+    echo "Source your .env.local or export the variable before running." >&2
+    exit 1
+fi
+
+# Create a temp directory for the script execution
+TEMP_DIR=$(mktemp -d)
+trap "rm -rf $TEMP_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)